src/main/java/uk/gov/hmcts/reform/logging/filters/RequestIdsSettingFilter.java

```diff
@@ -26,11 +26,9 @@ public RequestIdsSettingFilter(Supplier<String> requestIdGenerator) {
         this.requestIdGenerator = requestIdGenerator;
     }
 
-    @Override
     public void init(FilterConfig filterConfig) throws ServletException {
     }
 
-    @Override
```

Comment:
any reason for removing those?

Reply:
yes:

- they do not override anything. just implements interface definitions
- codacy not smart enough to recognise it being just a useless tag and complains about docblock being empty

Reply:
it's a good practice to use `@Override` when possible.

Reply:
As for codacy complaining: Is it a PMD rule? Is there a pmd file in this repo?

Reply:
as a rule of thumb you should not use override if you simply not overriding anything. as for doc inheritance, don't know why, but after some tab clicking and reaching the codebase i bluntly forgot about it :confused: 

please provide me a decent link where it says it's a good practice to use override for interface declared methods

Reply:
Every answer here for example: https://stackoverflow.com/questions/212614/should-we-override-an-interfaces-method-implementation

The cleared the code the better, won't you agree?

Also intelliJ adds those automatically when you create a new class that implements given interface.

Reply:
My preference would be to leave them as they were - they inform the developer that the method they're looking at is a part of an interface implementation. Not a huge value, of course, but still helpful.

Reply:
fair enough. since it is allowed from java 1.6 i'll add back. also will see what happens with doc inheritance tag.

my personal preference though still stays as logically i don't see the other way